Cargo.toml

```diff
@@ -32,16 +32,16 @@ autobenches = false
 [dependencies]
 approx = "0.5"
 itertools = "0.10"
-nalgebra = { version = "0.29", default-features = false, features = ["rand", "std"] }
+nalgebra = { version = "0.31", default-features = false, features = ["rand", "std"] }
 num-traits = "0.2"
 rayon = "1.3"
 tracing = { version = "0.1", default-features = false, features = ["std"] }
 rand = "0.8"
 sprs = { version = "0.11", default-features = false, features = ["multi_thread"] }
 
 [dev-dependencies]
-criterion = "0.3"
+criterion = "0.4"
 proptest = { version = "1.0.0", default-features = false, features = ["std", "timeout"] }
 tracing-subscriber = "0.3"
-tracing-chrome = "0.4"
+tracing-chrome = "0.6"
```

Comment:
The tracing-chrome update add a lot of deps, I opened a PR on their repo

https://github.com/thoren-d/tracing-chrome/pull/12